zhaquirks/develco/emi_led.py

```diff
@@ -25,6 +25,12 @@ class AttributeDefs(BaseAttributeDefs):
             type=t.uint16_t,
             is_manufacturer_specific=True,
         )
+        current_summation: Final = ZCLAttributeDef(
+            id=0x0301,
+            type=t.uint48_t,
+            access="r",
```

Comment:
But this should be write-only:
```suggestion
            access="w",
```